<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Myhouse</class>
 <widget class="QWidget" name="Myhouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	border-image: url(:/pictur/bg_home.jpg);
	
	border-image: url(:/pictur/bg_home1.jpg);
}
#frame_2{
	
	background-color: rgba(0, 0, 0, 100);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	background-color: rgba(255, 255, 255, 100);
	color: rgb(0, 0, 0);
border: 2px solid rgb(153, 230, 230);
border-radius:5px;
margin-top:2.1ex;
font-family:宋体;
font:bold 30px;
color: ;
	color: rgb(0, 247, 119);
}
 
QGroupBox::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>211</width>
       <height>331</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox{
	color: rgb(0, 255, 255);
}
QPushButton{
	background-color: rgba(255, 255, 255, 150);
	border-radius:10px; 
}
QPushButton:hover{
	background-color: rgba(85, 255, 255, 200);
    border:2px solid rgba(85, 255, 255, 200); 
    border-radius:10px; 
    color:white;
}
QPushButton:pressed{
	color: green;
	border-color: blueviolet;
	border-radius:10px; 
    background-color: white;
}</string>
     </property>
     <property name="title">
      <string>卧室</string>
     </property>
     <widget class="QPushButton" name="pushButton_fan1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictur/fan_close.png);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lamp1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictur/lamp_close.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>211</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_2{
	color: rgb(0, 255, 255);
}
QPushButton{
	background-color: rgba(255, 255, 255, 150);
	border-radius:10px; 
}
QPushButton:hover{
	background-color: rgba(85, 255, 255, 200);
    border:2px solid rgba(85, 255, 255, 200); 
    border-radius:10px; 
    color:white;
}
QPushButton:pressed{
	color: green;
	border-color: blueviolet;
	border-radius:10px; 
    background-color: white;
}
</string>
     </property>
     <property name="title">
      <string>客厅</string>
     </property>
     <widget class="QPushButton" name="pushButton_lamp2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictur/lamp_close.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fan2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictur/fan_close.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:10px; 
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 150);
    border:2px solid rgba(85, 255, 255, 200); 
    border-radius:10px; 
    color:white;
}
QPushButton:pressed{
	color: green;
	border-color: blueviolet;
	border-radius:10px; 
    background-color: white;
}</string>
     </property>
     <property name="title">
      <string>中央空调</string>
     </property>
     <widget class="QPushButton" name="pushButton_air">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictur/air_close.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>380</y>
       <width>221</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 10px;
	background-color: rgb(255, 255, 255);

font-family: &quot;Microsoft YaHei&quot;;
font-size: 14px;
font-style: italic;
font-weight: bold;
color: rgb(130, 130, 130);
font: bold italic 18px &quot;Microsoft YaHei&quot;;

padding-left: 7px;
padding-top: 7px;
padding-right: 7px;
padding-bottom: 7px;

}

QSpinBox {


	padding-left: 5px;
    padding-right: 15px;
    border-radius: 3px;

	color: rgb(0, 0, 0);
    background-color:rgb(222, 222, 222);
	font-family: &quot;Microsoft Yahei&quot;;
	font-size: 10pt;
}

QSpinBox:hover {
    background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <property name="title">
      <string>环境监控</string>
     </property>
     <widget class="QSlider" name="verticalSlider_humidity">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QSlider::groove:vertical {
border: 1px solid #4A708B;
background: #C0C0C0;
width: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
padding-top:-1px;
padding-bottom:-1px;
}

QSlider::sub-page:vertical {
background: rgb(255, 255, 255);
border: 1px solid #4A708B;
border-radius: 2px;
}

QSlider::add-page:vertical {

background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #c4c4c4, stop:1 #B1B1B1);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #5DCCFF, stop: 1 #1874CD);
background: rgb(130, 255, 134);
border: 0px solid #777;
width: 10px;
border-radius: 2px;
}

QSlider::handle:vertical 
{

    height: 11px;
    margin-left: -3px;
    margin-right: -3px;
    border-radius: 5px;
}

QSlider::sub-page:vertical:disabled {
background: #00009C;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: rgb(130, 255, 134);
border-color: #999;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_temperature">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {
border: 1px solid #4A708B;
background: #C0C0C0;
width: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
padding-top:-1px;
padding-bottom:-1px;
}

QSlider::sub-page:vertical {
background: #575757;
border: 1px solid #4A708B;
border-radius: 2px;
}

QSlider::add-page:vertical {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #c4c4c4, stop:1 #B1B1B1);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #5DCCFF, stop: 1 #1874CD);
border: 0px solid #777;
width: 10px;
border-radius: 2px;
}

QSlider::handle:vertical 
{

    height: 11px;
    margin-left: -3px;
    margin-right: -3px;
    border-radius: 5px;
}

QSlider::sub-page:vertical:disabled {
background: #00009C;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_humidity">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_temperature">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">




</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>湿度</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>470</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="title">
      <string>总开关</string>
     </property>
     <widget class="QPushButton" name="pushButton_fanTotal">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开风扇</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lampTotal">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>开灯</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
